### PR TITLE
Bug 1839465 - document how to manually apply a dep-signed mar file

### PR DESCRIPTION
**Pull request by jcristau**

### CODE REVIEW

**Review by bhearsum — APPROVED** [review]

File: how-to/update-testing.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Next, run the patched updater against the target binary::
+
+  wget https://archive.mozilla.org/pub/firefox/releases/114.0/win64/en-US/Firefox%20Setup%20114.0.exe
+  mkdir target
+  7z x -otarget "Firefox Setup 114.0.exe"
+  LD_LIBRARY_PATH=$PWD/firefox $PWD/update/updater $PWD/update $PWD/target/core/ $PWD/target/core/ 0
+  echo $?
+
+Finally check that the target directory looks as expected.
```
Comment (gbrownmozilla) [inline how-to/update-testing.rst:46]: I checked distribution.ini and channel-prefs.js -- all worked as expected for me. Thanks!

File: how-to/update-testing.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  wget https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/afYMhkhXSw6PsG0dWthmUA/runs/0/artifacts/public%2Fbuild%2Fswitch-to-esr115.0-eol-win.mar
+  cp public%2Fbuild%2Fswitch-to-esr115.0-eol-win.mar update/update.mar
+
+Next, run the patched updater against the target binary::
+
+  wget https://archive.mozilla.org/pub/firefox/releases/114.0/win64/en-US/Firefox%20Setup%20114.0.exe
+  mkdir target
+  7z x -otarget "Firefox Setup 114.0.exe"
+  LD_LIBRARY_PATH=$PWD/firefox $PWD/update/updater $PWD/update $PWD/target/core/ $PWD/target/core/ 0
```
Comment (gbrownmozilla) [inline how-to/update-testing.rst:43]: I found it unsettling that there was no output when running this; is there any way of coaxing status messages or something out of updater?

**Review by gbrownmozilla — APPROVED** [review]

File: how-to/update-testing.rst (hunk 2 above)
Comment (jcristau) [reply how-to/update-testing.rst:43]: Good point, would `cat update/update.log` after running the updater help?

Comment (gbrownmozilla) [reply how-to/update-testing.rst:43]: Yes, I think it is worth mentioning.

**Pull request merged**